src/rest/Facet/FacetRequest.ts

```diff
@@ -152,4 +172,17 @@ export interface IFacetRequest {
    * **Default (Search API):** `false`
    */
   generateAutomaticRanges?: boolean;
+
+  /**
+   * The character to use to split field values into a hierarchical sequence.
+   *
+   * **Example:**
```

Comment:
Ah, I know I reviewed the Search API Swagger docs from which this example was taken, but now that I look at it I find it extremely confusing :/

Do both ways work with the new API? I.e., is `c;folder2;folder3` equivalent to `c;c|folder2;|c|folder2|folder3;`?

Reply:
We could simplify it!
I don't really understand your question though why would they be equivalent?

Reply:
As discussed, since we plan to eventually remove support for multi-value fields containing hierarchically formatted metadata, let's juste eliminate the first part of the example, i.e., 

   * For a multi-value field containing the following values: `c; c&gt;folder2; c&gt;folder2&gt;folder3;`
   * The delimiting character is `&gt;`.